### DISCUSSION TITLE
VS Code ou Visual Studio?

### DISCUSSION THREAD

**Discussion by GeovaniDarcie** [balta-io/forum#337] [Tira Dúvidas] [answered]
Na minha empresa estou usando Visual Studio é um projeto MVC, a minha dúvida é se posso usar vs code normal, por conta das solutions, etc, e se isso iria impactar em alguma coisa...

E se é preferível usar um ou outro mesmo...

Pessoalmente eu uso os dois as vezes, mas não sei se isso impacta em grandes projetos

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @GeovaniDarcie, como vai?

Pode usar o Code sim, mas se vocês tem licença do VS, acredito que ele seja mais robusto.
Em relação a SLN, o Code trabalha diferente do VS, mas dá pra usar ele tranquilamente.
